pkgs/by-name/mu/mud-git/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  versionCheckHook,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mud";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jasursadikov";
+    repo = "mud";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-pW4B4+RN7hKtG2enJ33OHBeGsLj8w20ylvjcOL6owAk=";
+  };
+
+  build-system = with python3Packages; [
+    hatchling
+  ];
+
+  dependencies = with python3Packages; [
+    prettytable
+  ];
+
+  nativeCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = [ "--version" ];
+
+  meta = {
+    description = "Mud is a multi-directory git runner which allows you to run git commands in a multiple repositories";
```

```suggestion
    description = "multi-directory git runner which allows you to run git commands in a multiple repositories";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description